src/combine.cpp

```diff
@@ -4196,8 +4186,24 @@ static void mark_function(chunk_t *pc)
                isa_def = false;
                break;
             }
-            LOG_FMT(LFCN, " <skip %s>", prev->text());
-            prev = chunk_get_prev_ncnlnp(prev);
+#ifdef DEBUG
+            LOG_FMT(LFCN, "%s(%d):", __func__, __LINE__);
+#endif
+            LOG_FMT(LFCN, " <skip '%s'>", prev->text());
+#ifdef DEBUG
+            LOG_FMT(LFCN, "\n");
+#endif
+            // Issue #1112
+            prev = chunk_get_prev_ncnlnpnd(prev);
```

Comment:
i feel something wrong here, we are trying to skip words before DC_MEMBER, so why is DC_MEMBER being skiped?
i ran the new code with "uncrustify::name::func();" this line becomes a FUNC_PROTO instead of FUNC_CALL.

Reply:
I cannot reproduce.
Please push your configuration file and the output with
uncrustify ... -p p.txt